src/antsibull_docs_parser/parser.py

```diff
@@ -165,10 +169,19 @@ def _parse_option_like(
         text = text[len(_IGNORE_MARKER) :]
     else:
         plugin_identifier = context.current_plugin
+        entrypoint = context.role_entrypoint
+    if plugin_identifier is not None and plugin_identifier.type == "role":
+        idx = text.find(":")
+        if idx >= 0:
+            entrypoint = text[:idx]
+            text = text[idx + 1 :]
+        if entrypoint is None:
```

Comment:
Should this be

```suggestion
        if not entrypoint:
```
? I assume you'd also want a value like `:abc` (`text[:idx]` would be an empty string) to cause an error.

Reply:
I don't know whether ansible-core allows empty entrypoints. (Considering how flexible roles are in general, I wouldn't be surprised if it does.)